src/fitnesse/slim/SlimSymbol.java

```diff
@@ -0,0 +1,81 @@
+package fitnesse.slim;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+abstract public class SlimSymbol {
+  public static final Pattern SYMBOL_PATTERN = Pattern
+      .compile("\\$([A-Za-z\\p{L}][\\w\\p{L}]*)");
+// This would be a better pattern as it allows to define the end of a symbol name with another $ sign  
```

Comment:
The discussion for this commented code is in #780.

Yes the table would fail if you remove the new pattern. You can try with an older version of Fitnesse to prove it.

I will update the typos.